source/NationalInstruments/jquery.flot.historybuffer.analogWaveform.js

```diff
@@ -147,6 +147,17 @@ console.log(hb1.toDataSeries()); //[[4, 1], [5, 2], [6, 3], [null, null], [1, 1]
         }
     }
 
+    function appendWaveformSampleToDecimateBuffer(aw, sampleIndex, buffer) {
+        var Y = aw.Y,
+            TS = aw.t0,
```

Comment:
Why is TS upper case and not either ts or timeStamp?

Reply:
I was simply using the style that was already there. I, agree, I should go ahead and update these.

Reply:
Updated.